modules/hosted-control-planes-overview.adoc

```diff
@@ -11,7 +11,7 @@
 {hcp-capital} is available by using the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#cluster_mce_overview[{mce} version 2.0 or later] on the following platforms:
 
 * Bare metal by using the Agent provider
-* Non-bare metal Agent machines
+* Non-bare-metal Agent machines, as a Technology Preview feature
```

Comment:
Per ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens#compound-words
and SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_b

Changing `Non-bare metal` to `Non-bare-metal`